tools/karma_template.conf.js

```diff
@@ -77,8 +77,11 @@ const CUSTOM_LAUNCHERS = {
   chrome_webgpu: {
     base: 'ChromeCanary',
     flags: [
+      '--disable-dawn-features=disallow_unsafe_apis',
+      '--flag-switches-begin',
       '--enable-unsafe-webgpu',
-      '--disable-dawn-features=disallow_unsafe_apis'
+      '--enable-features=Vulkan',
```

Comment:
WebGPU on Linux requires command-line flag `--enable-features=Vulkan,UseSkiaRenderer`. So `UseSkiaRenderer` is missed? And this flag is only needed for Linux. We shouldn't add it on Windows. Can we distinguish them based on platforms?

Reply:
Thanks for the review! I added a [select](https://bazel.build/concepts/platforms-intro#select) to make sure the flag is added only on Linux. I also added `UseSkiaRenderer`.